src/magneticod/persistence.go

```diff
@@ -226,3 +227,30 @@ func setupSqliteDatabase(database *sql.DB) error {
 
 	return nil
 }
+
+func setupMySQLDatabase(database *sql.DB) error {
+	_, err := database.Exec(
+		`CREATE TABLE IF NOT EXISTS torrents ("
+			id		INTEGER PRIMARY KEY AUTO_INCREMENT,
+			info_hash	BINARY(20) NOT NULL UNIQUE,
+			name		VARCHAR(255) NOT NULL,
```

Comment:
It calculcates with characters since 5.x, and calculated in bytes before that. 

I'd prefer a `VARCHAR` because it is stored inline with the rest of the row, while types like `TEXT` and `BLOB` are just referenced and stored elsewhere.

We can just increase the size of the VARCHAR if 255 Unicode characters is not enough. I couldn't find a more appropriate value when skimming the relevant BEPs.